sdk/ml/azure-ai-ml/azure/ai/ml/entities/_builders/spark.py

```diff
@@ -199,7 +200,8 @@ def __init__(
             self.dynamic_allocation_max_executors = (
                 self.dynamic_allocation_max_executors or component.dynamic_allocation_max_executors
             )
-
+        if self.executor_instances is None and self.dynamic_allocation_enabled in ["True", "true", True]:
+            self.executor_instances = self.dynamic_allocation_min_executors
```

Comment:
could you point me to doc on this?

Reply:
@wangchao1230,  it's because douglas added this assignment statement 
![image](https://user-images.githubusercontent.com/26239730/222616450-2a2be480-56d1-4d10-b9d2-3e614313fc39.png)
 in _validate_spark_configurations and spark will call it when _to_rest_object. 
Since we no need add _validate_spark_configurations  in to_rest_function for now according to the pr description. So I add this statement here